.github/workflows/integration-json.yml

```diff
@@ -0,0 +1,48 @@
+name: Draft JSON Integration Tests
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18.2
+      - name: make
+        run: make
+      - uses: actions/upload-artifact@v2
+        with:
+          name: helm-skaffold
+          path: ./test/skaffold.yaml
+          if-no-files-found: error
+      - uses: actions/upload-artifact@v2
+        with:
+          name: draft-binary
+          path: ./draft
+          if-no-files-found:
+            error
+  info:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
+        with:
+          name: draft-binary
+      - run: chmod +x ./draft
+      - run: |
+          mkdir ./langtest
+      - run: |
+          ./draft info > ./info.json
+          echo "Draft Info JSON schema:"
+          cat test/info_schema.json
+            echo "Draft Info JSON:"
+          cat info.json
+      - name: Validate JSON
+        run: |
+          npm install -g ajv-cli
```

Comment:
💡 Thought share: I wonder we should lock this to specific version like `5.0.0` although there is no new release since 2 years, worth to make sure if repo owners published breaking changes it will not default flow into this repo.

🐈 Contrary: Although to counter this thought I could say that might be it is best to use latest so that we can capture any breaking changes early. 

I left both the thoughts for you to decide, but at least it Willa archive the reason we chose one way over other. thank you.

Reply:
this is a great point, thanks!